tensorbay/cli/auth.py

```diff
@@ -80,3 +94,11 @@ def _update_profile(config_parser: ConfigParser, profile_name: str, arg1: str, a
     config_parser["profiles"][profile_name] = (
         form_profile_value(arg1) if not arg2 else form_profile_value(arg2, arg1)
     )
+
+
+def _check_args_and_options(arg1: str, arg2: str, get: bool, is_all: bool) -> None:
+    if is_all and not get:
+        error('Use "--all" option with "--get" option')
```

Comment:
the `--all` flag can be used with `--get` `--unset` or `--status`
do we need to update the error message?

Reply:
The error message will be updated when `--unset` or `--status` is implemented.